doc/developers/contributing.rst

```diff
@@ -280,8 +280,8 @@ You can also check for common programming errors with the following tools:
     * Code with a good unittest coverage (at least 90%, better 100%), check
       with::
 
-        $ pip install nose coverage
-        $ nosetests --with-coverage path/to/tests_for_package
+        $ pip install pytest pytest-cov
+        $ pytest --cov sklearn path/to/tests_for_package
```

Comment:
Should we add an example to test a single test?

```
pytest path/to/tests_for_package::test_unit_test
```

Reply:
I'd rather try to keep this PR minimal, essentially the doc should only be a nose -> pytest translation.